src/components/atoms/Loader.tsx

```diff
@@ -2,14 +2,18 @@ import React, { ReactElement } from 'react'
 import styles from './Loader.module.css'
 
 export default function Loader({
-  message
+  message,
+  spinnerOnly
 }: {
   message?: string
+  spinnerOnly?: boolean
```

Comment:
question(blocking): Are we sure this didn't break anything as you mentioned before?
The safe change would have been to set this to true by default, so we do not have to adjust existing Loaders where this would be the expected behaviour of the component, right?

Reply:
edit: I see you went through the codebase adjusting this where necessary. Still: It is difficult to tell if there are possible cases that were missed and the workload was likely unecessary. At least for future adjustments like this, we need to make sure to try to stay away from introducing any new behaviour to components that require adjustments in the whole codebase.

Reply:
I went for this option because imho it looks more natural to pass a flag to modify some behaviour rather than pass a  false value. 
I do agree on the reasoning behind your comment, so I've changed them back, but I still think the false default is easier to read

Reply:
Updated the component to have an optional `useDefaultMessage` flag (false by default) instead of `spinnerOnly`